src/redux/types.js

```diff
@@ -0,0 +1,7 @@
+export const GET_CARS = 'GET_CARS';
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGIN_FAIL = 'LOGIN_FAIL';
+export const SIGNUP_SUCCESS = 'SIGNUP_SUCCESS';
+export const SIGNUP_FAIL = 'SIGNUP_FAIL';
+export const LOGOUT = 'LOGOUT';
+export const FETCH_GREETING = 'messages/FETCH_GREETING';
```

Comment:
## Hi @MudasirSherwani :wave: 

Great job for adding the Login and Sign up pages. However,  I do not understand why you have defined these constants.
- GET_CARS
- FETCH_GREETING

**Thank You**